MAINTAINERS

```diff
@@ -113,6 +113,11 @@ HiSilicon D02
 S:	Orphan
 F:	core/arch/arm/plat-d02/
 
+HiSilicon D06
+R:	zengxx <zengxiaoxu@huawei.com> [@xiaoxuZeng]
```

Comment:
We expect full names as "Firstname Lastname", that is: `Xiaoxu Zeng` rather than `zengxx` if you don't mind. Same in `Signed-off-by:` and git Author.

Reply:
Nice advice. thank you.